chef-config/lib/chef-config/config.rb

```diff
@@ -386,6 +386,19 @@ def self.path_accessible?(path)
     default :ez, false
     default :enable_reporting, true
     default :enable_reporting_url_fatals, false
+
+    ###
+    # Insights
+    #
+    # Insights is a product that integrates analytics, metrics and data for Chef,
+    # Delivery Workflow, and more. Currently included in Delivery licenses, this
+    # product is in open early access beta. Please contact tasha@chef.io with
+    # any product questions or suggestions.
+    #
+    # Sending data to Insights is disabled when URL is nil.
```

Comment:
Yeah, I think we just need the first part of the first sentence, "Insights is a product that integrates analytics, metrics and data for Chef." along with "Sending data to Insights is disabled when URL is nil."